charms/istio-pilot/config.yaml

```diff
@@ -1,4 +1,12 @@
 options:
+  cni-bin-dir:
+    type: string
+    default: ''
+    description: Path to CNI binaries, e.g. /opt/cni/bin. This configuration option will only work when first deploying istio.
```

Comment:
If we've [added `upgrade_charm` to reconcile](https://github.com/canonical/istio-operators/pull/365/files#diff-d3a4e684bd1a3a1822df5ab7605cfce00c97cd7b52f44ed770526c4f4f80eacdR293), is this statement still true?  I'm guessing that as soon as we change this, there will be a reconcile that redeploys istio with this setting.

Reply:
Correct, the message will be different here. I will update it to reflect what is actually happening. Good catch!

Reply:
Updated.